### ISSUE TITLE
Generate synthetic negative data

### ISSUE THREAD

**Issue by 0x000011b** [PygmalionAI/data-toolbox#11]
To test the implementation of the CRINGE loss in our training code, we need some examples of what the model should _not_ generate.

I have some filters in the `data-toolbox` that drop training examples based on certain criteria (e.g.: messages are too similar to each other indicating looping, or messages are too short on average). If we add a flag to generate using _only_ these dropped examples, we can build a training set of negative examples that we can use to test.

**Comment by TearGosling** [comment]
Very old issue, closing for now.
